### PR TITLE
Be a little less OMPI-centric on checking for the top-level directory

### PR DESCRIPTION
**Pull request by rhc54**
Signed-off-by: Ralph Castain <rhc@open-mpi.org>

### CODE REVIEW

File: contrib/update-my-copyright.pl [hunk 1]
```diff
@@ -120,18 +120,18 @@ sub quiet_print {
 $year += 1900;
 quiet_print "==> This year: $year\n";
 
-# Find the top-level OMPI source tree dir
+# Find the top-level repository source tree dir
 my $start = cwd();
 my $top = $start;
-while (! -f "$top/Makefile.ompi-rules") {
+while (! -f "$top/VERSION" && ! -f "$top/HACKING") {
```
Comment (jsquyres) [inline contrib/update-my-copyright.pl]: Do you want to look for `.git`?  That would make this work in any git repo (i.e., be even less OMPI-centric).

Comment (rhc54) [inline contrib/update-my-copyright.pl]: sure, that sounds fine

**Comment by rhc54** [comment]
done!

**Comment by jsquyres** [comment]
👍   Might as well merge; the CI doesn't test this script.

**Comment by rhc54** [comment]
yeah, i forgot to tell it to ignore

**Pull request merged**